src/DataType/NullHandler.php

```diff
@@ -10,13 +10,8 @@ class NullHandler extends ScalarHandler
     /**
      * {@inheritdoc}
      */
-    protected $type = 'NULL';
-
-    /**
-     * {@inheritdoc}
-     */
-    public function getDataType(): string
+    public function getType(): string
     {
-        return 'null';
+        return 'NULL';
```

Comment:
similar concerns here. Will we still pick up the right handler to unserialize with the different casing?

Reply:
`getType` returns the old `this->type`